src/SearchResults.js

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import moment from "moment";
+import { useState } from "react";
+import CustomerProfile from "./CustomerProfile";
+
+const SearchResults = props => {
+  const [color, setColor] = useState("transparent");
+  const [customerId, setCustomerId] = useState("");
+
+  function highlightRow(index) {
+    setColor(index === color ? "" : index);
+  }
```

Comment:
I'm not sure the naming here is quite right - it looks like you're setting the "color" state to be the index of the row that was clicked on.

How could you improve this?